src/components/CustomerQuota/AddUserModal.tsx

```diff
@@ -137,7 +137,7 @@ export const AddUserModal: FunctionComponent<AddUserModal> = ({
           <IdScanner
             onBarCodeScanned={onBarCodeScanned}
             onCancel={() => setShouldShowCamera(false)}
-            cancelButtonText="Enter NRIC manually"
+            cancelButtonText="Enter identification number manually"
```

Comment:
Can check if this will wrap on smaller phone screens? For mine it's from one end of the screen to the other already, concerned about overflow

Reply:
okay! trying to test on android emulator for this, looks damn long on my iphone also

Reply:
okay tested on a very small emulator, looks bad haha, spoke to Eida and changing it to "Enter ID manually" for now! 
![long](https://user-images.githubusercontent.com/43963814/84566655-8fae5f00-ada5-11ea-8dfb-90d434c23a63.png)